### PR TITLE
kitty-themes: 2022-05-04 -> 2022-08-11

### PR DESCRIPTION
**Pull request by supermarin**
###### Description of changes
Update kitty-themes. This adds some extra themes like Github Dark.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/kitty-themes/default.nix [hunk 1]
```diff
@@ -7,7 +7,7 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "kovidgoyal";
     repo = pname;
-    rev = "fb48041b0ff5ce60e8f10e7067a407ad99a4862e";
+    rev = "72cf0dc4338ab1ad85f5ed93fdb13318916cae14";
     sha256 = "/JCLty73YHsTkNxZP6EwhhoiHi2HjtyMZphAPhHe5h0=";
```
Comment (supermarin) [inline pkgs/misc/kitty-themes/default.nix:11]: getting the new sha

**Comment by vojta001** [comment]
Result of `nixpkgs-review pr 186552` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>kitty-themes</li>
  </ul>
</details>

**Comment by bobby285271** [comment]
Please also update `version`.

**Comment by supermarin** [comment]
@bobby285271 done

**Comment by bobby285271** [comment]
@ofborg eval

**Comment by NelsonJeppesen** [comment]
Can you please merge all these commits into one?

**Comment by bobby285271** [comment]
I can do a squash merge. Yeah for trivial update one commit for one package is enough, some committers prefer to sign their merge / some PRs are not suitable for squash merge (like those that touches more than one package) and merging maybe blocked in that case so it is always a good idea to fixup the change when possible.

**Pull request merged**